creating_gs_app.md

```diff
@@ -45,7 +59,7 @@ Ensure `rosjava` is installed:
 
 Build the ff_msgs jar
 
-    you@machine:~ $ cd $BUILD_PATH
+    you@machine:~ $ cd $BUILD_PATH/bulid/ff_msgs
```

Comment:
typo in `build`?

Reply:
yup

Reply:
Fixed in my latest [commit](https://github.com/nasa/astrobee_android/pull/48/commits/8aee4d09f352e1a4d1261a0cdce15ddeb420f43e)